karrot/groups/models.py

```diff
@@ -63,8 +57,21 @@ def annotate_yesterdays_member_count(self):
         )
 
 
+class GroupManager(models.Manager):
+    def create(self, *args, **kwargs):
+        try:
+            kwargs['theme'] = kwargs.get('theme', settings.GROUP_THEME_DEFAULT.value)
```

Comment:
That was a misunderstanding, it's definitely better to always have it present in settings.py instead of catching AttributeError.

Reply:
Okay. I changed it again.

The reason why I don't want to write something like

```
default=themes.GroupTheme.FOODSAVING
```

in ``models.py`` is, because it should be immediately clear which default value is actually used (settings.py or models.py), so everything, including the manager in models.py refers back to settings explicitly.